### PR TITLE
Start wiring up GraphQL to API

### PR DESCRIPTION
**Pull request by smith**
WIP

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -1,5 +1,41 @@
 module.exports = {
   plugins: [
+    {
+      resolve: `gatsby-source-apiserver`,
+      options: {
+        enableDevRefresh: true,
+        name: "assets",
+        url: "https://api.adoptopenjdk.net/v3/assets",
```
Comment (gdams) [inline gatsby-config.js]: It would probably be good to make the URL a parameter so we can change it easily if needed

Comment (smith) [reply gatsby-config.js]: Yes I'll probably do that before merge.

File: gatsby-config.js [hunk 2]
```diff
@@ -1,41 +1,63 @@
+const apiUrl = "https://api.adoptopenjdk.net/v3";
+
+// Available releases are hard-coded here to allow for the assets endpoint
+const availableVersions = [8, 9, 10, 11, 12, 13];
```
Comment (gdams) [inline gatsby-config.js]: You should be able to get this from https://api.adoptopenjdk.net/v3/info/available_releases

Comment (smith) [reply gatsby-config.js]: You can, but you can't make async requests in the config file. Still figuring out the best way to get this...

**Pull request closed without merging**